README.md

```diff
@@ -3,5 +3,688 @@
 Markdown Runner is a simple Go application designed to execute Markdown files as
 pipelines.
 
-You can use it to turn your Markdown documentation into executable tutorials, or
-into CI pipelines.
+One of the main use cases of the Markdown runner tool is to have your
+documentation testable in a CI environment. This way, you can always detect if
+your documentation is drifting away from the reality of your codebase.
+
+## Usage
+
+Installation on your system:
+
+```bash
+go install
+```
+
+Alternatively, you can run the tool directly with:
+
+```bash
+go run main.go --markdown-dir docs
+```
```

Comment:
Those instructions only works in the markdown-runner folder/project. Can you provide instructions for other folders/projects?